### PR TITLE
Add migration removing `binary`, `binary_components`, `is_restart_required` from `files`

### LINKED ISSUE #17682: create db migration to drop is_restart_required column
follow on from #17390 that must land in a subsequent tag/deploy to #17675

### LINKED ISSUE #17810: Remove `binary_components` and `binary` columns from the `files` table
https://github.com/mozilla/addons-server/issues/17792 will remove the `binary_components` and `binary` fields from the `Files` model, we then need to create the database migration dropping the columns them in a follow-up push.

### PR DESCRIPTION
**Pull request by diox**
Fixes #17682
Fixes #17810

### CODE REVIEW

**Review by eviljeff — APPROVED** [review]

**Comment by diox** [comment]
Since we want to deploy the CDN this week, I'll wait until after this week's tag to merge this - even with a single `ALTER TABLE` statement it's likely deployment duration would be affected, don't want to make this week's push even more annoying for ops.

**Pull request merged**
